tokio-console/src/state/mod.rs

```diff
@@ -538,3 +538,64 @@ fn pb_duration(dur: prost_types::Duration) -> Duration {
     let nanos = u64::try_from(dur.nanos).expect("duration should not be negative!");
     Duration::from_secs(secs) + Duration::from_nanos(nanos)
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    #[cfg(target_os = "linux")]
+    fn test_format_location() {
+        let mut location1 = proto::Location::default();
+        location1.file = Some(
+            "/home/user/.cargo/registry/src/github.com-1ecc6299db9ec823/tokio-1.0.1/src/lib.rs"
+                .to_string(),
+        );
```

Comment:
style nit, take it or leave it: i think we could also express this as
```suggestion
        let location1 = proto::Location {
            file: Some(
                "/home/user/.cargo/registry/src/github.com-1ecc6299db9ec823/tokio-1.0.1/src/lib.rs"
                    .to_string(),
            ),
            ..Default::default()
        };
```
which avoids the `mut` — it's entirely up to you whether you think this style is nicer, though.

Reply:
Good suggestion!